frontend/model/contracts/shared/time.js

```diff
@@ -128,7 +129,10 @@ export function humanDate (
     : ((navigator.languages: any): string[]) ?? navigator.language
   // NOTE: `.toLocaleDateString()` automatically takes local timezone differences into account.
   // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString
-  return new Date(date).toLocaleDateString(locale, options)
+  if (dateOrTime) {
+    return new Date(date).toLocaleDateString(locale, options)
+  }
+  return new Date(date).toLocaleTimeString(locale, options)
```

Comment:
I'm pretty sure these changes to the `humanDate` function can (& should) be undone because they aren't necessary.

The `toLocaleDateString` supports displaying the current time, you simply need to pass in the options telling it to, e.g. here's from me experimenting in the terminal:

```js
> new Date().toLocaleDateString('en-US', { month: 'short', day: 'numeric', hour: 'numeric', minute: '2-digit' })
'Apr 12, 4:46 PM'
```

Reply:
Okay. Sure.